fragments/training/training-fragment.md

```diff
@@ -1,6 +1,57 @@
 # Model Training Fragment
 
-**THIS SECTION IS STILL UNDER DEVELOPMENT.**
+This document describes the structure and content of a Model Training object.
 
 | Field Name          | Type                | Description                                                           |
 |---------------------|---------------------|-----------------------------------------------------------------------|
+| `created_at`        | string              | **Required.** Approximate date and time at which the model was trained. It is formatted according to [RFC 3339, section 5.6].  |
+| `environment`       | [Environment]       | **Required.** Description of the environment used to train the model. |
+| `source_imagery`    | [string]            | **Required.** A list of URLs, each of which points to a [STAC Collection] representing source imagery used to train the model. |
+| `labels`            | [string]            | A list of URLs, each of which points to a [STAC Collection] representing labels used to train the model. This is only applicable to supervised models and should be omitted for all unsupervised models. |
+
+#### created_at
+
+This field is intended to capture the approximate date and time at which the model was trained in order to give 
+other developers a sense of the currency of the model. The model publisher is free to choose the exact event that 
+this timestamp represents (e.g. start of model training, completion of model training, etc.). Publishers may also 
+choose to use `00` for all time values if they wish to only represent a date (e.g. `2020-02-01T00:00:00Z`).
```

Comment:
I would suggest to use just date without the time (e.g., `2020-02-01`) when publisher only wants to represent a date.

Reply:
That's a good point. I was trying to keep this consistent with the usage of RFC 3339 format in the STAC spec, but I agree it's awkward (and possibly confusing) here.

Reply:
I agree about awkwardness, but if STAC is following RFC 3339, then your suggestion is better, and comes with less possible issues down the road.

Reply:
Add a justification for this in 56fca86c739af329ec7aff76489a867008e62094 so it's clear to future users why we're doing this.